samples/nrf9160/nrf_cloud_mqtt_multi_service/src/main.c

```diff
@@ -8,50 +8,43 @@
 #include <zephyr/logging/log.h>
 #include "application.h"
 #include "connection.h"
+#include "led_control.h"
 #include "fota_support.h"
 
 LOG_MODULE_REGISTER(main, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
 
-static void msg_thread_fn(void)
```

Comment:
It's less that I removed post-fixes and more that I removed the entire `xxx_thread_fn` functions in favor of the functions they each were originally wrapping.

I could definitely add the wrappers back in -- I thought perhaps they were too verbose. But then again, they are a very concise way of explaining what is going on (We're starting a thread, it calls X function in another file).

Alternatively, `animate_leds`, `main_application`, `consume_device_messages`, `manage_connection` could be renamed to `led_thread_fn`, app_thread_fn`, `msg_thread_fn`, `conn_thread_fn`, but then the functions would be named after what they are, rather than what they do, which I think would be equally harmful to the clarity of the sample

Third option would be to explicitly mention the called function in each of the comments something like:
> Define, and automatically start the LED management thread and execute animate_threads. See led_control.c

I think I favor bringing the wrappers back, but I'm curious what you think in light of the above

Reply:
animate_leds_thread_fn
main_app_thread_fn
consume_msgs_thread_fn
conn_mgmt_thread_fn
?

Reply:
The problem with those is that they don't match up with the actual thread names: `led_thread`, `msg_thread`, `conn_thread`, `app_thread`

I could change the thread names, but `consume_messages_thread` seems strange to me

Reply:
I think it is worth the slightly awkward naming to follow your suggestion, but I'll need to think about what to rename the threads to

Reply:
this is not something to lose any sleep over.